platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateManager.java

```diff
@@ -304,6 +304,31 @@ public synchronized void handlePreconsensusSignatureTransaction(
         }
     }
 
+    /**
+     * An observer of post-consensus state signatures.
+     *
+     * @param signerId    the node that created the signature
+     * @param transaction the signature transaction
+     */
+    public synchronized void handlePostconsensusSignatureTransaction(
+            @NonNull NodeId signerId, @NonNull StateSignatureTransaction transaction) {
+
+        Objects.requireNonNull(signerId);
+
+        final long round = transaction.getRound();
+
+        try (final ReservedSignedState reservedState = getIncompleteState(round)) {
+            // it isn't possible to receive a post-consensus signature transaction for a future round,
```

Comment:
It is possible if this node is slow and has not yet reached consensus on a round that other nodes have. We should store signatures for a certain number of rounds in the future and only discard these if they are too far ahead. When a new state is created by this node, apply any signatures for it that were previously collected.

Reply:
@poulok If the signature transaction for a previous round is being handled by `nodeX`, isn't it guaranteed that the round the signature is on has _also_ already come to consensus for `nodeX`?

Reply:
ahh, yes of course, you are correct.